src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    public void doWork() {
+        System.out.println("Truck start work");

```

Comment:
The message 'Truck start work' could be more informative. It should indicate that it is a Truck that is working. Consider rephrasing it to include the type of machine, for example, 'Truck is working now.'